src/list/_list.scss

```diff
@@ -0,0 +1,90 @@
+@import "../variables";
+@import "../mixins";
+
+
+.mdl-list {
+  display: block;
+  padding: 8px 0 ;
+  font-family: $preferred_font;
+  list-style: none;
+}
+
+
+.mdl-list__item {
+  @include typo-body-1();
+  display: flex;
+  box-sizing: border-box;
+  height: 48px;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  align-items: center;
+  padding : 16px 16px 20px 16px;
+  cursor: default;
+
+  & .mdl-list__item-sub-header {
+    padding: 0 0 0 16px;
+  }
+
+  & .mdl-list__item-primary-content {
+    order:0;
+    flex-grow: 2;
+
+    & a {
+      @include typo-body-1();
+      color: $default-item-text-color;
+      text-decoration: none;
+    }
+
+    & .material-icons {
+      font-size: 40px;
```

Comment:
Not sure we should be resetting the size of the icons. Is there a specific reason for this?